docs/auctiongui/hidden-options.md

```diff
@@ -0,0 +1,75 @@
+# Hidden Configuration Options
+
+There are some hidden configuration options that can resolve some potential issues with AuctionGUI+ on some setups, along with provide additional behavior to the plugin, that may be potentially dangerous.
+
+## Number Formatting
+We provide several options which allow you to change the way numbers are formatted in messages sent to players. These have been shown below:
+
+### Decimal Separation
+This setting can be used to change the character used for separating decimals in figures which may not be integers. This is useful for other languages, who may use different characters for decimal separation. (E.g: Spain uses the comma to separate decimals.)
+
+This can be changed by inserting the following setting in the [`config.yml`](https://pastebin.com/KiM3PjU7):
+```yaml
+numberFormat:
+  # The default value for this is "."
+  decimalSeparator: '.'
+```
+
+### Grouping Separation
+This setting can be used to change the character used for separating groups of numbers for larger numbers. (E.g: 1,000,000) This is useful for other languages, who may use different characters for grouping separation. (E.g: Spain uses the decimal point to separate such groups.)
+
+This can be changed by inserting the following setting in the [`config.yml`](https://pastebin.com/KiM3PjU7):
```

Comment:
This Pastebin links to the ShopGUIPlus default config.yml, not AuctionGuiPlus.

Reply:
Amended.